### PR TITLE
[Core][Common] Adding snap mount

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Addition of snap mount and unmount. Also handling its cleanup.

Fixes: #861

Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: common/ops/gluster_ops/mount_ops.py [hunk 1]
```diff
@@ -157,3 +157,78 @@ def wait_for_mountpoint_to_connect(self, mountpoint: str,
...
+                   node: str, path: str, excep: bool = True) -> dict:
+        """
+        Args:
+            server (str): Hostname or IP address
+            volname (str): Name of volume whose snap is to be mounted
+            snapname (str): Name of the snap which is to be mounted.
+            path (str): The path of the mount directory(mount point)
+            node (str): The node where snap mount is to be run
+            excep (bool): exception flag to bypass the exception if the
+                          volume status command fails. If set to False
```
Comment (nik-redhat) [inline common/ops/gluster_ops/mount_ops.py:171]: nit: snap mount

Comment (srijan-sivakumar) [reply common/ops/gluster_ops/mount_ops.py:171]: 🤔 didn't get you...you mean to suggest modifying the function name or docstring for path parameter ?

Comment (nik-redhat) [reply common/ops/gluster_ops/mount_ops.py:171]: The doc string says: `volume status command fails` rather it should be `snap mount`.

Comment (srijan-sivakumar) [reply common/ops/gluster_ops/mount_ops.py:171]: Ahh..my bad 😓

Comment (srijan-sivakumar) [reply common/ops/gluster_ops/mount_ops.py]: seems like this docstring error is present in other places too...For now I've corrected the ones inside the mount_ops. Have to check other docstrings in other ops later.

Comment (nik-redhat) [reply common/ops/gluster_ops/mount_ops.py]: Might be :thinking:

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**